util/test/test_calcrewards.py

```diff
@@ -889,7 +951,7 @@ def _nullOwners(
     chain_nft_tups = calcrewards._getChainNftTups(stakes, nftvols_USD)
 
     owners: Dict[int, Dict[str, Union[str, None]]] = {}
-    for (chainID, nft_addr) in chain_nft_tups:
+    for chainID, nft_addr in chain_nft_tups:
```

Comment:
Similar to above: Please revert this, and any small refactor changes. Why: outside the scope of this PR, and makes reviewing this PR harder. If you want to get them in, do a separate PR just for small refactorings, merge it to main, then merge main into here.